openupgrade_scripts/scripts/base_automation/17.0.1.0/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,36 @@
+---Models in module 'base_automation'---
+---Fields in module 'base_automation'---
+base_automation / base.automation          / _order                        : _order is now 'id' ('sequence')
+base_automation / base.automation          / description (html)            : NEW
+base_automation / base.automation          / log_webhook_calls (boolean)   : NEW hasdefault: default
+base_automation / base.automation          / record_getter (char)          : NEW hasdefault: default
+base_automation / base.automation          / webhook_uuid (char)           : NEW
```

Comment:
This one should be initialized one by one for not getting all the records the same UUID.